src/components/Menu/Menu.scss

```diff
@@ -0,0 +1,202 @@
+@media screen and (max-width: 1343px) {
+  .menu-bars {
+    position: fixed;
+    left: calc(50% - 134px);
+    bottom: 8px;
+  }
+  .menu-bars--isAdmin {
+    left: calc(50% - 168px);
+  }
+  .menu-bars--user {
+    @media (prefers-color-scheme: light) {
+      .menu-bars__switch {
+        background-color: #FFFFFF;
+        box-shadow: 0 16px 32px 0 rgba(0, 87, 255, 0.24);
+      }
+      .menu-bars__switch > .switch__icon {
+        color: #ff0069;
+      }
+    }
+    @media (prefers-color-scheme: dark) {
+      .menu-bars__switch {
+        background-color: #ff0069;
+        box-shadow: 0 6px 9px 0 rgba(0, 0, 0, 0.2);
+      }
+      .menu-bars__switch > .switch__icon {
+        color: #FFFFFF;
+      }
+    }
+  }
+  .menu-bars--user > .user-menu {
+    visibility: visible;
+    opacity: 1;
+    bottom: 8px;
+  }
+  .menu-bars--user > .admin-menu {
+    visibility: hidden;
+    opacity: 0;
+    bottom: 72px;
+  }
+  .menu-bars--user > .menu-bars__switch > .switch__icon--add {
+    visibility: hidden;
+    opacity: 0;
+    transform: rotate(360deg);
```

Comment:
Why not `transform: rotate(0)` isn't that just the same?

Reply:
Because the default / initial situation is rotate(0). To perform a transformation you need to have a different radiant.